modules/client/game/css/game.css

```diff
@@ -13,6 +13,48 @@
   }
 }
 
+.game-page {
```

Comment:
Rewrote the css so we don't have to manually call a javascript resizeColumns function whenever the window resizes.

We still have to scale the canvas image but at least we don't have to manually resize the canvas DOM element.